### DISCUSSION TITLE
Manually attaching and trimming a volume

### DISCUSSION THREAD

**Discussion by rasmusson** [longhorn/longhorn#8073] [Q&A]
I have a volume that is only attached under short periods of time as it is only used by a cron job. As mentioned in this discussion #8035 , I have trouble doing recurring trimming the volume as it is not attached. Im now working on a workaround where I manually attach the volume from the GUI and then perform trimming of that volume from the GUI. 

I can attach the volume and it get marked as healthy, but when try to trim I get the following error message. 
"unable to trim filesystem for volume pvc-0f7bb829-ff84-482d-af09-19c1ce8d4fde: volume frontend is disabled"

Is this a bug or is it just something I don't understand?

**Comment by derekbit** [comment]
Do you attach the volume with maintenance mode (`spec.disableFrontend=false`)?

**Comment by rasmusson** [comment]
Right right, I tought I got the same message with and without maint mode. Without main mode I get "failed to find valid mountpoint"

**Reply by derekbit** [reply]
Doesn't it work if you
- attach the volume to a node
- mount the block device /dev/longhorn/<volume name> to a mount point
- then, trim the mount point

**Comment by rasmusson** [comment]
Ok, can I mount it from the UI or in kubernetes or do I need to mount in using the mount command in the OS cli?

**Reply by derekbit** [reply]
Well, need a workload using the volume if you don't want to mount it by yourself.

**Reply by rasmusson** [reply]
Right, I solved this by deploying a cronjob that deploys a pod mounting the volume and then sleeping for 10 minutes. A recurring trim job is then set to start 2 min after this.
